install/gcp/installer/util/gcloud.py

```diff
@@ -51,6 +51,13 @@ def get_gcloud_info():
             sys.exit(1)
     return project_id, authed_user, is_devshell
 
+def check_network_host_project_id(self):
```

Comment:
This is doing more than just checking.  So, it would be better to say:
`set_network_host_project_id()`